### DISCUSSION TITLE
ORB Slam / VSLAM

### DISCUSSION THREAD

**Discussion by nombrote** [emgucv/emgucv#505] [General]
Hi, anybody have an implementation of slam with emgu? visual slam, ORB Slam, for example.

If not.. know here can i get it? 

Thanks

**Comment by emgucv** [comment]
Not in C# / Emgu, but you can use ROS (a sister project of Open CV)

https://towardsdatascience.com/ros-autonomous-slam-using-randomly-exploring-random-tree-rrt-37186f6e3568#:~:text=ROS%20and%20SLAM,will%20be%20used%20for%20exploring.

https://www.ros.org/

**Reply by nombrote** [reply]
Yes, i know ROS, and i know it has a non native connector on C# to any of their methods, but is dificult to find a full specification and a sample to use and try that algorithm.

**Reply by emgucv** [reply]
It is best to contact ROS developers for this question.

**Comment by serhatyaris34** [comment]
Hi, you can check out this senior project. It might inspire something for you.
https://youtu.be/Ii88mdvKZNA
